sdk/identity/identity/test/internal/node/managedIdentityCredential.spec.ts

```diff
@@ -397,15 +393,7 @@ describe("ManagedIdentityCredential", function() {
       );
 
       assert.equal(authRequest.headers.get("Authorization"), `Basic ${key}`);
-      if (authDetails.token) {
-        // We use Date.now underneath.
-        assert.equal(
-          Math.floor(authDetails.token.expiresOnTimestamp / 1000000),
-          Math.floor(Date.now() / 1000000)
-        );
-      } else {
-        assert.fail("No token was returned!");
-      }
+      assert.ok(authDetails.token?.expiresOnTimestamp);
```

Comment:
There’s really no reason to test the contents of expiresOnTimestamp. We define this value in our test utilities. The processing of the request will take a variable time depending on the environment (Node, VM, etc)